semantic_router/splitters/base.py

```diff
@@ -0,0 +1,12 @@
+from typing import List
+
+from pydantic.v1 import BaseModel
+from semantic_router.encoders import BaseEncoder
+
+class BaseSplitter(BaseModel):
+    name: str
+    encoder: BaseEncoder
+    similarity_threshold: float
```

Comment:
we should align naming conventions, in encoders/route layer this is called `score_threshold`

Reply:
Done!